### PR TITLE
fix(IDPay): [IODPAY-212] "Tap to scroll" floating button in IDPay onboarding screen

### PR DESCRIPTION
**Pull request by mastro993**
## Short description
This PR adds a "tap to scroll" button in IDPay initiative onboarding screen.

https://user-images.githubusercontent.com/6160324/232086854-6ab2c3ca-2d69-4338-9c89-c15fc0045e1f.mp4

## List of changes proposed in this pull request
- Added `ScaleInOutAnimation` custom animation 
- Added `ForceScrollDownView` which renders a scroll view with a button to scroll to the bottom
- Added skeleton to `InitiativeDetailsScreen`
- Added `ForceScrollDownView` to `InitiativeDetailsScreen`

## How to test
`yarn test:dev` and, with the `io-dev-server`, navigate to the onboarding screen of an IDPay initiative. Check that the "tap to scroll" button is working correctly.

### CODE REVIEW

File: ts/features/idpay/onboarding/components/ScrollDownButton.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import { StyleSheet } from "react-native";
+import Animated, { ZoomIn, ZoomOut } from "react-native-reanimated";
+import IconButtonSolid from "../../../../components/ui/IconButtonSolid";
+
+type Props = {
+  isVisible?: boolean;
+  onPress: () => void;
+};
+
+const ScrollDownButton = (props: Props) => {
```
Comment (hevelius) [inline ts/features/idpay/onboarding/components/ScrollDownButton.tsx:11]: could it be useful to add a test for this component? what do you think?

Comment (mastro993) [reply ts/features/idpay/onboarding/components/ScrollDownButton.tsx]: I've added a new component, `ForceScrollDownView`, which handles all the logic. This component is more testable. See https://github.com/pagopa/io-app/pull/4541/commits/59cfbd94ba39e03d767954bbc261f6ec8f04c11c

File: ts/components/ForceScrollDownView.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    <>
+      <ScrollView
+        testID={"ScrollView"}
+        ref={scrollViewRef}
+        scrollIndicatorInsets={{ right: 1 }}
+        scrollEnabled={scrollEnabled}
+        onScroll={handleScroll}
+        scrollEventThrottle={400}
+        style={style}
+        onLayout={handleLayot}
```
Comment (forrest57) [inline ts/components/ForceScrollDownView.tsx:184]:
```suggestion
        onLayout={handleLayout}
```
small typo, nothing exceptional

Comment (mastro993) [reply ts/components/ForceScrollDownView.tsx]: Fixed in https://github.com/pagopa/io-app/pull/4541/commits/2fc36d2b748f0ee18110b401b86065a809354703

**Review by forrest57 — APPROVED** [review]

**Pull request merged**